CONTRIBUTING.md

```diff
@@ -1,47 +1,11 @@
 # Contributing Guidelines
-* Subscribe to our [systers-dev](http://systers.org/mailman/listinfo/systers-dev) mailing list and send out an introductory email.
-* You can join our [slack channel](http://systers.io/slack-systers-opensource/).  Each active repo has its own channel to direct questions to (for example #powerup or #portal).  Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](https://github.com/systers/vms/blob/master/code_of_conduct.md).
-
-[General Guidelines](#general-guidelines)  
-[Opening a New Issue](#opening-a-new-issue)  
-[Important Reminders](#important-reminders)  
-[Communicate](#communicate)  
-[Don’t Know Code?](#dont-know-code)  
+* You can join our [slack channel](http://systers.io/slack-systers-opensource/).  Each active repo has its own channel to direct questions to (for example #powerup or #portal).  
+* Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](systers.io/code-of-conduct.html).
```

Comment:
why are these still here ?

Reply:
Could you share why you feel this shouldn't be here?

Reply:
@tapasweni-pathak got me wrong. why are these still here in the sense - these are not part of the commit. Possibly due to rebase issues. Check the PR title. `Volunteer's organization changed from none and organization list added in edit profile`